auth-api/src/auth_api/models/affiliation_invitation.py

```diff
@@ -105,15 +103,58 @@ def create_from_dict(cls, invitation_info: dict, user_id, affiliation_id=None):
         affiliation_invitation.save()
         return affiliation_invitation
 
+    # pylint: disable=too-many-arguments
     @classmethod
-    def find_invitations_by_sender(cls, user_id):
-        """Find all affiliation invitations sent by the given user."""
-        return cls.query.filter_by(sender_id=user_id).all()
+    def filter_by(cls,
```

Comment:
Use data class, this is way too many parameters.

Reply:
Added